cmd/cstor-pool-mgmt/pool/v1alpha2/disk_operation.go

```diff
@@ -120,15 +133,16 @@ func removePoolVdev(csp *apis.CStorPoolInstance, bdev apis.CStorPoolClusterBlock
 }
 */
 
-func replacePoolVdev(csp *apis.CStorPoolInstance, bdev apis.CStorPoolClusterBlockDevice, npath string) error {
-	if IsEmpty(npath) || IsEmpty(bdev.DevLink) {
+func replacePoolVdev(csp *apis.CStorPoolInstance, bdev apis.CStorPoolClusterBlockDevice, npath []string) error {
+	if len(npath) == 0 || IsEmpty(bdev.DevLink) {
```

Comment:
is bdev.DevLink doesn't stay empty all the time? are we updating CSPI CR with DevLink during pool creation or adding vdev group?

Reply:
No. During next update, It will get updated. In this PR, we are updating this during pool creation.